<launch>
    <arg name="shipname" default="Viknes830" />
    <group ns="$(arg shipname)">
        <node pkg="usv_mission_planner"
            name="mission_planner_client_node"
            type="mission_planner_client_node"
            respawn="false"
            output="screen"
            launch-prefix="">
        </node>
    </group>
</launch>